index.d.ts

```diff
@@ -0,0 +1 @@
+export function formatValue(value: number, unit: string): number;
```

Comment:
Nice, do you need to indicate in `package.json` where the TypeScript definitions are or is `index.d.ts` assumed?

Reply:
`index.d.ts` is assumed as the default. We can give it a different name and add an explicit reference in `package.json` if preferred but I thought this was cleaner.

Reply:
No need, default is good. I was just confirming.